examples/capture/getUserMedia.js

```diff
@@ -0,0 +1,71 @@
+// Resulotion Map
+// Map resolution option name to value
+const resolutionMap = {
+  "1080p": { width: 1920, height: 1080 },
+  "720p": { width: 1080, height: 720 },
+  "480p": { width: 854, height: 480 },
+  "360p": { width: 640, height: 360 },
+  "240p": { width: 426, height: 240 }
+}
+
+// Temasys Adapter JS
+// https://github.com/Temasys/AdapterJS
+// AdapterJS provides polyfills and cross-browser helpers for WebRTC. It wraps around
+// the native APIs in Chrome, Opera and Firefox and provides support for WebRTC in 
+// Internet Explorer and Safari on Mac and Windows through the available Temasys Browser Plugins.
+
+$W.getUserMedia = function(options) {
+  AdapterJS.webRTCReady(() => {
+    // The WebRTC API is ready.
+    const container = document.getElementById('webcam'),
+          video = document.createElement('video');
+    
+    container.appendChild(video);
+    video.autoplay = true;
+    video.id = 'webcam-video'
+    
+    const successCallback = stream => {
+      $('#heightIndicator').show()
+      $('#webcam-msg').hide()
+      attachMediaStream(video, stream)
+      if ($W.flipped == true) {
+        $W.flipped = false; // <= turn it false because f_h() will toggle it. messy.
+        $W.flip_horizontal();
+      }
+    };
+
+    const errorCallback = () => console.warn(error);
+
+    getUserMedia($W.defaultConstraints, successCallback, errorCallback);
+  });
+};
+
+$W.updateResolution = function(resolution) {
+  const width = resolutionMap[resolution].width,
+        height = resolutionMap[resolution].height;
+
+  const constraints = {
+    ...$W.defaultConstraints,
+    video: {
```

Comment:
I believe the `facingMode` constraint can be added here!

https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints/facingMode

```suggestion
    facingMode: { ideal: "environment" },
    video: {
```

Once this suggestion is accepted, it will need to be rebuilt to generate the compiled output file.